apps/api/src/test-data/buildEdgeConnectionTestData/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    EdgeConnection,
+    EdgeConnectionMemberRole,
+    EdgeConnectionType,
+} from '../../domain/models/context/edge-connection.entity';
+import { PageRangeContext } from '../../domain/models/context/page-range-context/page-range.context.entity';
+import { TextFieldContext } from '../../domain/models/context/text-field-context/text-field-context.entity';
+import { resourceTypes } from '../../domain/types/resourceTypes';
+
+export default (): EdgeConnection[] =>
+    [
+        {
```

Comment:
break this out into a separate file. You might consider having:
- buildSelfEdgeConnections
- buildDualEdgeConnections
As a strategy for organization.

Reply:
done